docs/source/documentation.rst

```diff
@@ -115,6 +115,8 @@ This integrity test automatically performs the axon and myelin segmentation of a
 
     * * * Integrity test passed. AxonDeepSeg is correctly installed. * * * 
 
+.. NOTE :: For some users, the test may fail because Keras is using Theano backend instead of Tensorflow. In that case, you will see the line ``Using Theano backend.`` when launching ``axondeepseg_test``. To fix this issue, add the line ``export KERAS_BACKEND="tensorflow"`` at the end of the ``<your_conda_install_location>\envs\<your_environment_name>/etc/conda/activate.d/keras_activate.sh`` file, then deactivate and reactivate your environment. The test should print ``Using Tensorflow backend.`` and pass.
+
```

Comment:
Do we need to warn about this?

IIRC, the Keras project has moved on from non-Tensorflow backends ever since it was integrated into TF. 

Though, now that I'm looking into this, they recently edited their README to [remove information about this](https://github.com/keras-team/keras/commit/4fd825ddf93d3dc9ecc7b50178ce1d015393bed2#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5). They used to have this warning:

> **Multi-backend Keras has been discontinued. At this time, we recommend that Keras users who use multi-backend Keras with the TensorFlow backend switch to `tf.keras` in TensorFlow 2.0**.

But now it's ["under construction"](https://github.com/keras-team/keras/blob/master/README.md). Huh.

Reply:
We decided to warn about this because in particular, one of our test users encountered exactly this issue here in this thread =)

Reply:
I guess what I'm saying is, could this be an isolated incident that doesn't apply to most users? :stuck_out_tongue: